src/frontend/js/modules/application-menu.js

```diff
@@ -1,131 +1,40 @@
-(function(angular) {
+const { applyPolyfills, defineCustomElements } = require('esn-frontend-application-grid/loader');
 
-  'use strict';
+applyPolyfills().then(() => defineCustomElements());
 
-  angular.module('esn.application-menu', [
-    'op.dynamicDirective',
-    'feature-flags'
-  ])
-
-  .constant('POPOVER_APPLICATION_MENU_OPTIONS', {
-    animation: 'am-fade',
-    placement: 'bottom',
-    template: require("../../views/modules/application-menu/application-menu.pug"),
-    html: false,
-    trigger: 'manual',
-    autoClose: true,
-    prefixEvent: 'application-menu',
-    container: 'body'
-  })
-  .constant('APP_MENU_OPEN_EVENT', 'application-menu.open')
-
-  .factory('applicationMenuTemplateBuilder', function(featureFlags, _) {
-    var template =
-    '<div>' +
-    '<a href="<%- href %>" target="<%- target %>" rel="<%- rel %>">' +
-    '<img class="esn-application-menu-icon" src="<%- iconURL %>" fallback-src="/images/application.png"/>' +
-    '<span class="label" translate>' +
-    '<%- label %>' +
-    '</span>' +
-    '</a>' +
-    '</div>';
-
-    var svgTemplate =
-      '<div>' +
-      '<a href="<%- href %>" target="<%- target %>" rel="<%- rel %>">' +
-      '<div class="esn-application-menu-icon" ng-include="\'<%- iconURL %>\'"></div>' +
-      '<span class="label" translate>' +
-      '<%- label %>' +
-      '</span>' +
-      '</a>' +
-      '</div>';
-
-      return function(href, icon, label, flag, isDisplayedByDefault) {
-        var iconURL, aHref, aTarget;
-        var aRel = '';
-      var iconUrlTemplate = '/images/application-menu/<%- icon %>-icon.svg';
-      var defaultValue = angular.isDefined(isDisplayedByDefault) ? isDisplayedByDefault : true;
-      var isActive = defaultValue;
-
-      if (angular.isDefined(flag)) {
-        isActive = featureFlags.isOn(flag) === undefined ? defaultValue : featureFlags.isOn(flag);
-      }
-
-      if (!isActive) {
-        return '';
-      }
-
-      if (icon.url) {
-        iconURL = icon.url;
-      } else if (icon.name) {
-        iconURL = _.template(iconUrlTemplate)({icon: icon.name});
-      }
-
-      if (angular.isObject(href)) {
-        aHref = href.url;
-        aTarget = href.target || '';
-        aRel = href.rel || aRel;
-      } else {
-        aHref = href;
-        aTarget = '';
-      }
-
-      return iconURL && iconURL.match(/.\S+\.svg/) ?
-      _.template(svgTemplate)({ href: aHref, target: aTarget, rel: aRel, iconURL: iconURL, label: label }) :
-      _.template(template)({ href: aHref, target: aTarget, rel: aRel, iconURL: iconURL, label: label });
-    };
-  })
-
-  .directive('applicationMenuToggler', function($rootScope, $document, $popover,
-    POPOVER_APPLICATION_MENU_OPTIONS, APP_MENU_OPEN_EVENT) {
+// TODO: Write tests for this
+angular.module('esn.application-menu', ['op.dynamicDirective'])
```

Comment:
Is `op.dynamicDirective` needed?

Reply:
No, it's no longer needed. I'll remove it.